x/gov/client/cli/tx.go

```diff
@@ -61,35 +59,93 @@ func NewTxCmd(propCmds []*cobra.Command) *cobra.Command {
 		RunE:                       client.ValidateCmd,
 	}
 
-	// TODO Add CLI for new submit proposal
-	// https://github.com/cosmos/cosmos-sdk/issues/10952
-	cmdSubmitProp := NewCmdSubmitProposal()
+	cmdSubmitLegacyProp := NewCmdSubmitLegacyProposal()
 	for _, propCmd := range propCmds {
 		flags.AddTxFlagsToCmd(propCmd)
-		cmdSubmitProp.AddCommand(propCmd)
+		cmdSubmitLegacyProp.AddCommand(propCmd)
 	}
 
 	govTxCmd.AddCommand(
 		NewCmdDeposit(),
 		NewCmdVote(),
 		NewCmdWeightedVote(),
-		cmdSubmitProp,
+		NewCmdSubmitProposal(),
+		cmdSubmitLegacyProp,
 	)
 
 	return govTxCmd
 }
 
-// NewCmdSubmitProposal implements submitting a proposal transaction command.
+// NewCmdSubmitLegacyProposal implements submitting a proposal transaction command.
 func NewCmdSubmitProposal() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "submit-proposal",
-		Short: "Submit a proposal along with an initial deposit",
+		Short: "Submit a proposal along with some messages and metadata",
+		Args:  cobra.ExactArgs(1),
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`Submit a proposal along with some messages and metadata.
+Messages, metadata and deposit are defined in a JSON file.
+
+Example:
+$ %s tx gov submit-proposal path/to/proposal.json
+
+Where proposal.json contains:
+
+{
+  // array of proto-JSON-encoded sdk.Msgs
+  "messages": [
+    {
+      "@type": "/cosmos.bank.v1beta1.MsgSend",
+      "from_address": "cosmos1...",
+      "to_address": "cosmos1...",
+      "amount":[{"denom": "stake","amount": "10"}]
+    }
+  ],
+  "metadata: "4pIMOgIGx1vZGU=", // base64-encoded metadata
+  "deposit": "10stake"
+}
```

Comment:
LMK what you think about this JSON format for submitting a proposal.

Reply:
Yeah I think that's preferable